docs/production/requirements.md

```diff
@@ -5,6 +5,22 @@ like, we recommend creating an account on the
 [Zulip community server](../contributing/chat-zulip-org.html), or creating a test
 organization on <https://zulipchat.com>.
 
+## Summary
+
+To run a Zulip server, you will need:
+* A dedicated machine or VM
+* A supported OS:
+  * Ubuntu 16.04 Xenial 64-bit
+  * Ubuntu 14.04 Trusty 64-bit (not recommended for new installations)
+* At least 2GB RAM, and 10GB disk space
+  * If you expect 100+ users: 4GB RAM, and 2 CPUs
```

Comment:
We might want this recommendation to include 50GB of space, since the database gets larger with more users.